Sources/SQLKit/Query/SQLQueryString.swift

```diff
@@ -119,6 +119,10 @@ extension SQLQueryString {
     public static func +(lhs: SQLQueryString, rhs: SQLQueryString) -> SQLQueryString {
         return "\(lhs)\(rhs)"
     }
+    
+    public static func +=(lhs: inout SQLQueryString, rhs: SQLQueryString) {
+        lhs.fragments.insert(contentsOf: rhs.fragments, at: lhs.fragments.startIndex)
```

Comment:
Whoops, my mistake - I got it a little backwards 🤦‍♀️ It should be:
```suggestion
        lhs.fragments.append(contentsOf: rhs.fragments)
```

Reply:
Oops, I should have checked that :)